include/state-observation/dynamics-estimators/kinetics-observer.hpp

```diff
@@ -254,10 +299,15 @@ class STATE_OBSERVATION_DLLAPI KineticsObserver : protected DynamicalSystemFunct
   /// @brief remove all the contacts
   void clearContacts();
 
+  /// @brief Get the Number Of Contacts
+  ///
+  /// @return Index The number of contacts
+  Index getNumberOfContacts() const;
+
   /// @brief Get the Current Number Of Contacts
   ///
   /// @return Index The current number of contacts
```

Comment:
Same doc as above but it does a different computation

Reply:
I removed getNumberOfContacts as it is not used and I cannot find a real use case for it

Reply:
I removed the getNumberOfContacts as I added it and it is not useful